static/app/components/performance/searchBar.spec.tsx

```diff
@@ -163,7 +163,7 @@ describe('SearchBar', () => {
 
     expect(screen.queryByTestId('smart-search-dropdown')).not.toBeInTheDocument();
     expect(onSearch).toHaveBeenCalledTimes(1);
-    expect(onSearch).toHaveBeenCalledWith('transaction:client*');
```

Comment:
Can we maybe also check the request + permutations being made via the api mock w/ the index test so we're extra sure the url's being created make sense?

Reply:
yeah good point. i'll add it in a follow up pr because it's a bit annoying to test since history only gets pushed to at the top most level so I have to test it there